bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -337,6 +339,112 @@ func (gcp *Gcp) ConfigK8s() error {
 	return nil
 }
 
+// Add a conveniently named context to KUBECONFIG.
+func (gcp *Gcp) AddNamedContext() error {
```

Comment:
Could we add a unittest to verify the config file is modified correctly?

Reply:
The integration test is good. 

It would be good to have a unittest as well; but we can leave that for a follow on PR since I think we have higher priority items for 0.5.

For a unittest we might want to refactor the code so that we have a simple method to modify the file. Then we can write a unittest in which we list test cases with input and expected output and verify that the code does the correct thing.

Reply:
sounds good, adding a new issue for tracking: #2817